my-portfolio/package.json

```diff
@@ -19,6 +19,7 @@
     "@angular/platform-browser": "~8.0.0",
     "@angular/platform-browser-dynamic": "~8.0.0",
     "@angular/router": "~8.0.0",
+    "p5": "^0.8.0",
```

Comment:
What does this p5 library do??

Reply:
It's from the processing org website, I use that to make all those codes that I want to add to my website.